Maddori.Apple/Maddori.Apple/Screen/Home/AddFeedback/AddFeedbackKeywordViewController.swift

```diff
@@ -0,0 +1,290 @@
+        }
+        
+        view.addSubview(containerScrollView)
+        containerScrollView.snp.makeConstraints {
+            $0.horizontalEdges.equalToSuperview().inset(SizeLiteral.leadingTrailingPadding)
+            $0.top.equalTo(keywordTextField.textFieldContainerView.snp.bottom).offset(32)
+            $0.bottom.equalTo(doneButton.snp.top).offset(-12)
+        }
+        
+        containerScrollView.addSubview(scrollContentStackView)
+        scrollContentStackView.snp.makeConstraints {
+            $0.verticalEdges.equalToSuperview().inset(8)
+            $0.horizontalEdges.equalToSuperview().inset(24)
+            $0.width.equalToSuperview().offset(-2 * SizeLiteral.leadingTrailingPadding)
+        }
+    }
+    
+    // MARK: - func
+    
+    override func setupNavigationBar() {
+        super.setupNavigationBar()
+        
+        let button = removeBarButtonItemOffset(with: backButton, offsetX: 10)
+        let backButton = makeBarButtonItem(with: button)
+        let closeButton = makeBarButtonItem(with: closeButton)
+        
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationItem.largeTitleDisplayMode = .never
+        navigationItem.leftBarButtonItem = backButton
+        navigationItem.rightBarButtonItem = closeButton
+    }
+    
+    private func setupNotificationCenter() {
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillShow), name: UIResponder.keyboardWillShowNotification, object: nil)
+        NotificationCenter.default.addObserver(self, selector: #selector(keyboardWillHide), name: UIResponder.keyboardWillHideNotification, object: nil)
+    }
+    
+    private func didTappedBackButton() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    private func didTappedCloseButton() {
+        self.dismiss(animated: true)
+    }
+    
+    private func setupTextFieldObserver() {
+        keywordTextField.keywordTextField.addTarget(self, action: #selector(changeDoneButtonStatus), for: .editingChanged)
+    }
+    
+    private func didTappedDoneButton() {
+        guard let keyword = keywordTextField.keywordTextField.text else { return }
+        let dto = FeedBackContentDTO(type: feedbackType, keyword: keyword, content: contentString, start_content: nil, to_id: toUserId)
+        dispatchAddFeedBack(type: .dispatchAddFeedBack(reflectionId: reflectionId, dto))
+    }
+    
+    // MARK: - api
+    
+    private func dispatchAddFeedBack(type: AddFeedBackEndPoint<FeedBackContentDTO>) {
+        AF.request(type.address,
+                   method: type.method,
+                   parameters: type.body,
+                   encoder: JSONParameterEncoder.default,
+                   headers: type.headers
+        ).responseDecodable(of: BaseModel<FeedBackContentResponse>.self) { json in
+            dump(json.value)
+            DispatchQueue.main.async {
+                self.dismiss(animated: true)
+            }
+        }
```

Comment:
p1;

이 부분에서 순차적인 처리가 안되는 것 같아요 !
API 통신이 제대로 이루어졌을 때 dismiss되어야 home 화면으로 갔을 때, 내가 작성한 키워드가 바로 보입니다 !
```suggestion
        ).responseDecodable(of: BaseModel<FeedBackContentResponse>.self) { json in
            if let data = json.value {
                dump(data)
                DispatchQueue.main.async {
                    self.dismiss(animated: true)
                }
            }
        }
```
if let 에서 json.value가 있다는 뜻이 통신이 제대로 이루어졌다는 뜻입니다 !

Reply:
[266307a](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/218/commits/266307ae8bfd86cb94663dfe79763ec076284f47)

엇 순서에 오류가 있었군요!! 감사합니다!!